src/helpers/getAddressToMerkle.ts

```diff
@@ -0,0 +1,41 @@
+import { QUERY_BLOCK_LIMIT } from '@big-whale-labs/constants'
+import { SealCredLedger } from '@big-whale-labs/seal-cred-ledger-contract'
+import AddressToMerkleRootMap from 'models/AddressToMerkleRootMap'
+
+export default async function (sealCredLedger: SealCredLedger) {
+  const setMerkleRootFilter = sealCredLedger.filters.SetMerkleRoot()
+  const deleteMerkleRootFilter = sealCredLedger.filters.DeleteMerkleRoot()
+
+  const topics = []
+
+  if (deleteMerkleRootFilter.topics)
+    topics.push(
+      Array.isArray(deleteMerkleRootFilter.topics[0])
```

Comment:
Can you explain why we need this check? Do we not know the return type?

Reply:
It could be string, array of string or undefined and I should extract only string

Reply:
In theory, it would be enough to specify `topics: [[deleteMerkleRootFilter.topics[0], setMerkleRootFilter.topics[0]]]`, but due to the fact that `deleteMerkleRootFilter.topics` may be `undefined` or be `string[]` you have to extract the value like this